CHANGELOG.md

```diff
@@ -18,7 +18,7 @@ The types of changes are:
 
 
 ## [Unreleased](https://github.com/ethyca/fidesops/compare/1.5.0...main)
-* 
+* Updated documentation for the user management ui [#469](https://github.com/ethyca/fidesops/issues/469)
```

Comment:
A new `Docs` section should be added to help organize the updates. Also the link should point to the pull request that is merging in the changes. 

I'll make an issue to update the checklist item to mention it should link to the PR instead of the issue. The PR is used so users can see the diff of what was merged in. 

```suggestion
### Docs
* Updated documentation for the user management ui [#530](https://github.com/ethyca/fidesops/pull/530)
```